conf/messages

```diff
@@ -94,12 +94,12 @@ guarantee.accessCode.title = What is the access code?
 guarantee.accessCode.heading = What is the access code?
 guarantee.accessCode.paragraph = This is set up by the Principal and works just like a bank PIN code.
 guarantee.accessCode.hint.transition = The code will be 4 characters long, like 0000 or X9X9.
-guarantee.accessCode.hint.postTransition = The code can be up to 4 characters long, like 0 or X9X9.
+guarantee.accessCode.hint.postTransition = The code will be 4 characters long, like 0000 or X9X9.
 guarantee.accessCode.checkYourAnswersLabel = Access code
 guarantee.accessCode.error.required = Enter the access code
 guarantee.accessCode.error.length.transition = The access code must be 4 characters long
 guarantee.accessCode.error.length.postTransition = The access code must be 4 characters or less
-guarantee.accessCode.error.invalid = The access code must only include the letters a to z and numbers 0 to 9
+guarantee.accessCode.error.invalid = The access code must be 4 characters long
```

Comment:
Hmm, not sure I agree with this. Looking at the content docs we still need the "The access code must only include the letters a to z and numbers 0 to 9" message when they enter invalid characters